ferry.go

```diff
@@ -583,7 +607,12 @@ func (f *Ferry) Run() {
 	}
 
 	binlogWg := &sync.WaitGroup{}
-	binlogWg.Add(2)
+
+	waitNum := 2
+	if !f.Config.SkipTargetVerification {
+		waitNum = 3
+	}
+	binlogWg.Add(waitNum)
```

Comment:
nit: can you `Add(1)` (right before each `go func...`) instead of having this conditional?